templates/jenkins-slave-defaults.erb

```diff
@@ -35,7 +35,7 @@ MAXOPENFILES=8192
 
 MASTER_URL="<%= @masterurl -%>"
 AUTO_DISCOVERY_ADDRESS="<%= @autodiscoveryaddress -%>"
-LABELS="<%= @_real_labels -%>"
+LABELS="<%= @labels.join(' ') -%>"
```

Comment:
Note that the plist entry doesn't have an entry for this, but I don't know the swarm config at all so I'm fine with it. If anyone needs it, a PR is welcome.